Large Upload Playbook (#32867)
demisto/content: Packs/CortexXDR/ReleaseNotes/6_1_22.md

#### Layouts

##### Cortex XDR Incident

> Added a dedicated tab for Large Upload alerts handling.

Reviewer: ```suggestion
- Added a dedicated tab for the "Large Upload" alert handling.
```

Author: done

#### Playbooks

##### New: Cortex XDR - Large Upload

- The playbook investigates Cortex XDR incidents involving large upload alerts. The playbook is designed to run as a sub-playbook of ‘Cortex XDR Alerts Handling v2’. 

The playbook consists of the following procedures:
- Searches for similar previous incidents that were closed as false positives.
- Enrichment and investigation of the initiator and destination hostname and IP address.
- Enrichment and investigation of the initiator user, process, file, or command if it exists.
- Detection of related indicators and analysis of the relationship between the detected indicators.
- Utilize the detected indicators to conduct threat hunting.
- Blocks detected malicious indicators.
- Endpoint isolation.

This playbook supports the following Cortex XDR alert names:
- Large Upload (Generic)
- Large Upload (SMTP)
- Large Upload (FTP)
- Large Upload (HTTPS) (Available from Cortex XSOAR 6.10.0).
##### Cortex XDR Alerts Handling v2

> Added the 'Cortex XDR - Large Upload' sub-playbook.

Reviewer: ```suggestion
- Added the 'Cortex XDR - Large Upload' sub-playbook.
```

Author: done

##### Cortex XDR - Search and Compare Process Executions - XDR Alerts

> Updated the playbook description.

Reviewer: ```suggestion
- Updated the playbook description.
```

Author: done
